rr_viz/package.xml

```diff
@@ -19,6 +19,7 @@
   <build_depend>rospy_message_converter</build_depend>
   <build_depend>rviz_textured_sphere</build_depend>
   <build_depend>rr_diagnostics</build_depend>
+  <build_depend>rr_custom_msgs</build_depend>
 
 
   <run_depend>actionlib</run_depend>
```

Comment:
I can't comment at the exact line but "message_runtime" can probably be removed from here too.

Reply:
@msadowski  I just tried to build without the message_runtime line and it failed so will leave it in

Reply:
this is very odd since it does not generate messages. I will create a jira issue for this so that we can check this later on.